pavelib/paver_tests/test_paver_bok_choy_cmds.py

```diff
@@ -6,7 +6,12 @@
 
 import os
 import unittest
-from test.test_support import EnvironmentVarGuard
+import six
+
+if six.PY3:
+    from test.support import EnvironmentVarGuard   # pylint: disable=import-error,no-name-in-module
+else:
+    from test.test_support import EnvironmentVarGuard
```

Comment:
Python 2.7 also has `test.support.EnvironmentVarGuard`, so we shouldn't need a condition for this: https://docs.python.org/2/library/test.html#module-test.support

Reply:
That's what I had originally but the version of python we're currently running(2.7.12) doesn't have that port yet so I had to fall back to this.

Reply:
Ok, then could we at least switch this to an `if six.PY2:` condition?  The way it's currently written is likely to break if there's an eventual `six.PY4`.